### PR TITLE
End2end test modelzoo releasing model definition

### PR DESCRIPTION
**Pull request by typhoonzero**

### CODE REVIEW

File: pkg/modelzooserver/image_build.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	"os/exec"
+	"strings"
+)
+
+func imageExistsOnRegistry(imageName, tag string) bool {
+	var imageNamePart string
+	var registryPart string
+	registryPart = os.Getenv("SQLFLOW_MODEL_ZOO_REGISTRY")
+	imageNamePart = imageName
+	if strings.Contains(imageName, ".") {
```
Comment (lhw362950217) [inline pkg/modelzooserver/image_build.go:31]: “." -> "/" ?

Comment (lhw362950217) [inline pkg/modelzooserver/image_build.go:33]: Are we using '.' to determine whether the name is start with a domain？Maybe add some comment

File: python/sqlflow_submitter/import_custom_models.py [hunk 2]
```diff
@@ -22,5 +22,6 @@ def import_model_def(estimator_name, namespace):
                 try:
                     namespace[model_name_parts[0]] = __import__(
                         model_name_parts[0])
+                    return model_name_parts[0]
```
Comment (lhw362950217) [inline python/sqlflow_submitter/import_custom_models.py]: shall we add some log if the estimator_name is not in "a.b" format? Or, we just intend to ignore it.

Comment (typhoonzero) [reply python/sqlflow_submitter/import_custom_models.py]: I think just ignore it will be fine.

**Review by lhw362950217 — APPROVED** [review]
LGTM

**Review by wangkuiyi — APPROVED** [review]

**Pull request merged**